docs/array-utils.md

```diff
@@ -14,3 +14,23 @@ async function callChunked<Item>(
 ```
 
 Splits `array` by passed `chunkSize` and run callback asynchronously for every chunk in a sequential order.
+
+
+
+`removeNullish<const T>(array: readonly (T | null | undefined)[]): T[]`
+
+Returns a copy of the given array without null or undefined values.
+```typescript
+const array = ['', false, null, 'valid', 1, undefined, 0]
+console.log(removeNullish(array)) // result: ['', false, 'valid', 1, 0]
+```
+
+
+
+`removeNullish<const T>(array: readonly (T | null | undefined)[]): T[]`
+
+Return a copy of the given array without falsy values (eg: false, 0, '', null, undefined).
+```typescript
+const array = ['', false, null, 'valid', 1, undefined, 0]
+console.log(removeNullish(array)) // result: ['valid', 1]
```

Comment:
```suggestion
`removeFalsy<const T>(array: readonly (T | null | undefined)[]): T[]`

Return a copy of the given array without falsy values (eg: false, 0, '', null, undefined).
```typescript
const array = ['', false, null, 'valid', 1, undefined, 0]
console.log(removeFalsy(array)) // result: ['valid', 1]
```

Reply:
Ups, I am sorry, and thank you so much Igor 🙇